### PR TITLE
Kerberos

### PR DESCRIPTION
**Pull request by ThierryO**

### CODE REVIEW

**Comment by ThierryO** [comment]
This seems stable enough to be merged into the master. Any objections? @stijnvanhoey?  @Jo-Loos? @florisvdh?

**Comment by stijnvanhoey** [comment]
Maybe we should get `inbo-sql07-prd.inbo.be` out of the documentation online? @Jo-Loos

**Comment by florisvdh** [comment]
Several comments made at 2066a8cf were not handled yet.

Further, the 'untested' paragraph on `odbc.ini` may need updates and cleaning to get the tutorial more or less finished.

I leave that to @stijnvanhoey to make decisions.

File: content/installation/user/kerberos.Rmd [hunk 1]
```diff
@@ -194,21 +198,21 @@ Beside the fact that the connection has been made (see RStudio's R console), you
...
 #### UNTESTED: Connecting after configuring `odbc.ini`
 
-However, it is probably easier to provide the configuration to specific databases directly, using the `/etc/odbc.ini` file. For example, the `NBNData_IPT` database can be defined as follows:
+However, it is probably easier to provide the configuration to specific databases directly, using the `/etc/odbc.ini` file. For example, the `DBName` database can be defined as follows:
 
 ```
 [nbn_ipt]
 Driver      = ODBC Driver 13 for SQL Server
-Description = INBOSQL07\PRD
+Description = odbc vebinding naar db
```
Comment (florisvdh) [inline content/installation/user/kerberos.Rmd]: small typo here

Comment (ThierryO) [reply content/installation/user/kerberos.Rmd]: fixed in 40f4336

**Pull request merged**